ex11/part5/src/colorblockbriten.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import tinycolor from './tinycolor';
```

Comment:
I know Ynon recommended TinyColor, however, I have a slightly different approach:
You don't need to use a special color library, in CSS you can define color in the HSL format:
Color, Hue, Saturation: https://css-tricks.com/hsl-hsla-is-great-for-programmatic-color-control/
As a rule of thumb, it is always better to write a few extra lines yourself than to add a package.